src/moonlink/src/storage/iceberg/rest_catalog_test.rs

```diff
@@ -0,0 +1,83 @@
+#[cfg(feature = "rest-catalog")]
+#[cfg(test)]
+mod tests {
+    use crate::storage::iceberg::rest_catalog::RestCatalog;
+    use crate::storage::iceberg::rest_catalog_test_guard::RestCatalogTestGuard;
+    use crate::storage::iceberg::rest_catalog_test_utils::*;
+    use iceberg::{Catalog, NamespaceIdent};
+
+    #[tokio::test(flavor = "multi_thread", worker_threads = 2)]
+    async fn test_create_table() {
+        let namespace = random_string();
+        let table = random_string();
+        let _guard = RestCatalogTestGuard::new(namespace.clone(), None)
+            .await
+            .expect("Namespace creation fail");
```

Comment:
```suggestion
            .expect("Rest catalog test guard creation fail");
```
namespace or table creation is an internal detail